samples/gtk/build.gradle.kts

```diff
@@ -0,0 +1,71 @@
+import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTargetPreset
+import org.jetbrains.kotlin.konan.target.KonanTarget.*
+
+plugins {
+    kotlin("multiplatform")
+}
+
+// Determine host preset.
+val hostOs = System.getProperty("os.name")
+
+val hostPreset: KotlinNativeTargetPreset = when {
```

Comment:
BTW, why don't we have this function in Gradle plugin?

Reply:
@ilmat192 would you add this to the backlog?

Reply:
Ok